src/Test/Utilities/Portable/CommonTestBase.cs

```diff
@@ -526,5 +528,269 @@ internal static SignatureDescription Signature(string fullyQualifiedTypeName, st
         }
 
         #endregion
+
+        #region Operation Test Helpers
+        internal static void VerifyParentOperations(SemanticModel model)
+        {
+            HashSet<string> output = new HashSet<string>();
```

Comment:
We do use pooled objects in a number of tests. That said, this helper is only used in two tests, so forget it.

---
In reply to: [130969619](https://github.com/dotnet/roslyn/pull/20964#discussion_r130969619) [](ancestors = 130969619)